README.md

```diff
@@ -108,6 +108,28 @@ Run project
   npm install
 ```
 
+### Environment Variables
+
+To run this project, you need to set up the following environment variables:
+
+- `REACT_APP_API_KEY`: Obtain this API key from [API Provider](https://api-ninjas.com/api/quotes) and store it in your `.env.local` file.
+- `REACT_APP_OTHER_KEY`: Similarly, if there are other keys required, mention them here.
```

Comment:
[Optional]👏 Excellent job on your project! I have a suggestion to improve the documentation for setting environment variables. Providing a comprehensive list of all the necessary environment variables along with descriptions can be incredibly helpful for users. 

It has been noticed in your project documentation that you have not provided information about the necessary environment variables. Currently, users have to manually search through each file to find out about each environment variable being used, which is not a good practice. Although you added this information to the project submission link for me, it may not be helpful for other users. To improve the user experience, it is recommended that you list all the required environment variables along with brief descriptions for each. This approach will make it easier for users to understand the purpose of each variable and ensure they provide the correct values.Here's an example of how it could look:

```plaintext
# API URL for fetching quotes
REACT_APP_QUOTES_URL=https://example.com/api/quotes

# API Key for accessing the quotes API
REACT_APP_QUOTES_KEY=your-api-key

# Category of quotes to retrieve (e.g., 'inspirational', 'funny', 'motivational')
REACT_APP_QUOTES_CATEGORY=inspirational
```

By including descriptions along with the variable names and values, you provide users with clear guidance on what each variable is used for. This can significantly enhance the user experience and help them set up the environment variables correctly. Keep up the great work in improving your project's usability! 👍📝🔐